.gitignore

```diff
@@ -7,4 +7,8 @@ target
 *.js
 *.js.map
 *.css
-*.css.map
\ No newline at end of file
+*.css.map
+
+# IDE specific
```

Comment:
Would you normally put this in a separate PR or are small changes like this allowed along side a feature PR?

Reply:
Here is absolutely fine :+1: